qhana_plugin_registry/__init__.py

```diff
@@ -137,6 +138,9 @@ def create_app(test_config: Optional[Dict[str, Any]] = None):
 
         register_debug_routes(app)
 
+    # populate endpoint attributes of the metadata
+    populate_endpoints()
```

Comment:
This call should be moved to the `qhana_plugin_registry/api/__init__.py` file into the `register_root_api` function.